odpf/raccoon/v1/raccoon.proto

```diff
@@ -0,0 +1,163 @@
+syntax = "proto3";
+
+package odpf.raccoon.v1;
+
+import "google/protobuf/timestamp.proto";
+
+option java_multiple_files = true;
+option java_package = "io.odpf.proton.raccoon";
+option java_outer_classname = "EventProto";
+
+option go_package = "github.com/odpf/proton/raccoon/v1;raccoonv1";
+
+
+
+service EventService {
+  rpc SendEvent (EventRequest) returns (EventResponse);
```

Comment:
@ravisuhag @chakravarthyvp need your inputs here, buf lint is recommending the change of name for EventRequest and EventResponse. This will be a breaking change, so shall we ignore this or make the change? Another thing we can do is probably have both `SendEventRequest/Response` and `EventRequest/Response` with same fields for now and then slowly phase`EventRequest/Response`out. What do you folks think?

Reply:
It should not break since this changing just the names and not the tag numbers. I suggest we verify this commit in one of our integration and wire an old version proto against it.

Reply:
The `EventRequest` is independent of GRPC and IMO we should keep it that way. However, `SendEventRequest` sounds like a good name even from a WebSocket or HTTP perspective. I don't mind renaming.

Reply:
+1 @NNcrawler 

@ramey - Are the EventRequest/Reponse/Event moved out of the service proto? I see you upvoted @NNcrawler 's comment.

We can keep the gRPC service proto as `v1beta`

Reply:
No @chakravarthyvp I have kept everything in the same proto file from readability point of view. If renaming doesn't break things then let's rename them.